guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc

```diff
@@ -2,7 +2,7 @@
 = Editing the System Purpose of a Host
 
 You can edit the system purpose attributes for a {RHEL} host.
-System purpose attributes define which subscriptions to attach automatically to this host.
+System purpose attributes define which subscriptions to attach automatically to the host.
```

Comment:
Auto-attach is only relevant when SCA is turned off.

I would suggest using the text Rich Jerrido gave me for the tooltip:
_System purpose allows you to set the system's intended use on your network and improves the reporting of usage in Subscription Watch._